examples/ex6p.cpp

```diff
@@ -2,19 +2,19 @@
 //
 // Compile with: make ex6p
 //
-// Sample runs:  mpirun -np 4 ex6p -m ../data/star.mesh -o 3
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 1 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2
+// Sample runs:  mpirun -np 4 ex6p -m ../data/star-hilbert.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 1
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2 -h1
```

Comment:
I think we should keep at least one `-cs` sample run -- we want to test that part of the code too.

Reply:
There's `../data/escher.mesh -o 2 -cs`, do you want to add more?

Reply:
That's 3D (tets). I was referring to the above 2D (triangles) sample runs.

Reply:
Ah, OK. I guess the third run could be `-rm 1 -o 2 -cs` (right now it's too similar to the second run anyway).